### DISCUSSION TITLE
Use Gray Code as Encoding.

### DISCUSSION THREAD

**Discussion by RollingTechSupport** [ahmedfgad/GeneticAlgorithmPython#264] [Q&A]
I started working with pygad and I'm really impressed by it's functionalities. After first trials and studying of the documentation I wonder if there is a way to use gray code as the encoding type?

**Comment by ahmedfgad** [comment]
Hi @ChRen95,

Thanks for your interest. Unfortunately, only decimal and binary encodings are supported by PyGAD!

You can still support gray encoding but you have to put some efforts into building other functionalities.
